gubernator/view_build.py

```diff
@@ -52,10 +52,9 @@ def handle_test(self, child, filename, name_prefix=''):
         elif child.find('failure') is not None:
             time = float(child.attrib['time'])
             out = []
-            for param in child.findall('system-out'):
-                out.append(param.text)
-            for param in child.findall('system-err'):
-                out.append(param.text)
+            for param in child.findall('system-out') + child.findall('system-err'):
+                if param.text is not None:
```

Comment:
How do `None` and `''` differ?

Reply:
'\n'.join([None]) doesn't work

Reply:
Do we explicitly want to add empty strings?